plugins/woocommerce/src/Internal/CLI/Migrator/Commands/ProductsCommand.php

```diff
@@ -51,24 +61,53 @@ final public function init( CredentialManager $credential_manager, PlatformRegis
 	 * [--count]
 	 * : Only fetch and display the total product count.
 	 *
-	 * [--fetch]
-	 * : Fetch and display product data from the platform.
-	 *
 	 * [--limit=<limit>]
-	 * : Maximum number of products to fetch (default: 5).
-	 *
-	 * [--after=<cursor>]
-	 * : Pagination cursor for fetching products after a specific point.
+	 * : Maximum number of products to migrate.
 	 *
 	 * [--status=<status>]
 	 * : Filter products by status (active, archived, draft).
 	 *
+	 * [--product-type=<product-type>]
+	 * : Filter products by type (for Shopify: any product type name, or 'single'/'variable' for WooCommerce equivalents).
+	 *
+	 * [--vendor=<vendor>]
+	 * : Filter products by vendor name.
+	 *
+	 * [--ids=<ids>]
+	 * : Comma-separated list of product IDs to migrate.
+	 *
+	 * [--batch-size=<size>]
+	 * : Number of products to process per batch (default: 20, max: 250).
+	 *
+	 * [--fields=<fields>]
+	 * : Comma-separated list of fields to migrate.
+	 *
+	 * [--exclude-fields=<fields>]
+	 * : Comma-separated list of fields to exclude from migration.
+	 *
+	 * [--resume]
+	 * : Resume from previous migration session without prompting.
```

Comment:
Is it worth mentioning here that `--resume` will not preserve arguments from the previous command?

Example:
- I run `wp wc migrate products --fields=name,price,images`
- I walk away from computer while migration takes place, and some interuption happens.
- I try `wp wc migrate products --resume` and expect the i will resume the previous command exactly as it was.